frontend/src/components/UserHome/UserHome.tsx

```diff
@@ -68,32 +77,39 @@ const UserHome: React.FC<{
           address(es). <Link route={{ type: "profile" }}>Check</Link>
         </Alert>
       )}
-
-      {data.primaryEmail ? (
```

Comment:
There is a [snapshot test](https://github.com/matrix-org/matrix-authentication-service/blob/1fe9a34260f7e69759193036d6b624a4d276ab6e/frontend/src/components/UserHome/UserHome.test.tsx#L104) and a [story](https://github.com/matrix-org/matrix-authentication-service/blob/1fe9a34260f7e69759193036d6b624a4d276ab6e/frontend/src/components/UserHome/UserHome.stories.tsx#L126-L135) just for this, could you either restore this alert or remove the corresponding tests?